Extensions/PlatformBehavior/platformruntimebehavior.ts

```diff
@@ -229,6 +229,19 @@ namespace gdjs {
     static NORMALPLAFTORM = 0;
     static JUMPTHRU = 1;
     static LADDER = 2;
+
+    static isOnPlatformTest(object1, object2, behaviorName) {
+      // Check if the objects exist and share the behavior
+      if (object1 === null || !object1.hasBehavior(behaviorName)) {
```

Comment:
I think object1 is guaranteed to be not null, same as object2.

Reply:
Actually the behavior is also guaranteed by the action calling it, so no need to check.